pkg/learning/exec_result.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package learning
+
+type ExecResult struct {
```

Comment:
This is reasonable to move to mab_ss.go because it's only used for MABSeedScheduler and is specific to MABSeedScheduler. File per type is generally excessive.

Reply:
I originally put it here so the planned TaskScheduler can use it as well. But perhaps it's better to move it to SeedScheduler and move it out later if needed.